### PR TITLE
OpenStack UPI: Create API and Ingress ports

### PR DESCRIPTION
**Pull request by mandre**
This creates the API and Ingress ports and attaches the user provided
FIPs to these newly created ports.

### CODE REVIEW

File: upi/openstack/inventory.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ all:
       os_flavor_worker: 'm1.large'
       os_image_rhcos: 'rhcos'
       os_external_network: 'external'
+      os_api_fip: ''
+      os_ingress_fip: ''
```
Comment (pierreprinetti) [inline upi/openstack/inventory.yaml]: I am not sure what the user should insert here

Comment (mandre) [reply upi/openstack/inventory.yaml]: This should be the floating IP addresses that were created in step https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md#api-and-ingress-floating-ip-addresses.

Comment (pierreprinetti) [reply upi/openstack/inventory.yaml]: Do you think it's worth adding a small comment stating that?

File: upi/openstack/02_network.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      - subnet: "{{ nodes_subnet }}"
+        ip_address: "{{ os_subnet_range | next_nth_usable(7) }}"
+
+  # NOTE: openstack ansible modole doesn't let you attach a FIP to a port, we
+  # have to use the CLI instead
+  - name: 'Attach the API floating IP to API port'
+    command:
+      cmd: "openstack floating ip set --port {{ os_port_api }} {{ os_api_fip }}"
+
+  # NOTE: openstack ansible modole doesn't let you attach a FIP to a port, we
```
Comment (pierreprinetti) [inline upi/openstack/02_network.yaml]: oh and while you're here, there's this modole (2 occurrences)

**Comment by pierreprinetti** [comment]
/lgtm

Once you see `openssl -nodes`, you can't unsee it :smile:

**Pull request merged**